tools/webpack

```diff
@@ -46,6 +49,17 @@ def build_for_dev_server(host, port, minify, disable_host_check):
     if disable_host_check:
         webpack_args.append('--disable-host-check')
 
+    cwd = os.getcwd()
```

Comment:
This section probably deserves a block comment explaining what it's all about.

Reply:
Added.